src/components/app_catalog/catalog_list/CatalogList.js

```diff
@@ -23,7 +23,7 @@ class CatalogList extends React.Component {
               resources yet for your installation. Please come back later!
             </p>
           ) : (
-            <React.Fragment>
+            <>
```

Comment:
Is this one really needed? I see only one child

Reply:
Yeah doesn't look like it needs to be there. Removing.